ingress/loadbalancer_util.go

```diff
@@ -0,0 +1,39 @@
+package ingress
+
+import (
+	log "github.com/Sirupsen/logrus"
+)
+
+// Functions that benefit from isolated testing
```

Comment:
Maybe it would make sense to move LoadBalanceryEntry in here as well, and rename the file `lb_entry.go` or something like that. Then we keep the data structure with the functions that operate on it.